agent/util-scripts/pbench-postprocess-tools

```diff
@@ -155,7 +155,7 @@ for this_tool_file in `/bin/ls $tool_group_dir`; do
 		debug_log "[${script_name}] ${pbench_bin}/tool-scripts/${name} --${action} --dir=${tool_output_dir} ${tool_opts[@]}"
 		if [ "$action" == "start" ]; then
 			# using screen to avoid tty issues and guarantee tool is backgrounded
-			screen_dir="${tool_output_dir}/.screen.d/${group}-${name}"
+			screen_dir="${tool_output_dir}/.screen.${hostname}.d/${group}-${name}"
```

Comment:
Is this `hostname` variable going to have the same value as the `hostname` variable used when the directory is moved?

Reply:
And won't the `.screen.${hostname}.d` not be copied by both the local `mv` and remote `tar` command use a simple `*` wildcard and not referencing the directory?

Can't wait for the Tool Meister (#1248) to fix this.